util/find-doc-nits

```diff
@@ -255,6 +255,20 @@ sub check {
         if $contents =~ /=over([^ ][^24])/;
     err($id, "Possible version style issue")
         if $contents =~ /OpenSSL version [019]/;
+    err($id, "Brackets on item line")
+        if $contents =~ /=item \[/;
+    if ( $contents !~ /=for comment generic/) {
+        # Some API pages have B<foo<I<TYPE>bar>.
+        err($id, "Bad flag formatting inside B<>")
+            if $contents =~ /B<-[A-Za-z_ ]+ /;
+        while ( $contents =~ /([BI])<([^>]*)>/g ) {
+            my $B = $1;
+            my $T = $2;
+            next if $T =~ /E</;  # Assume it's E<lt>
+            err($id, "Bad content inside $B<$T>")
+                if $T =~ /[<|]/;
+        }
+    }
```

Comment:
You is checking the *whole* content, right?  That's gonna hurt...  see, DESCRIPTION is free flow text, and both bold and italics are allowed to be used for *emphasis*.  Sure, there are conventions on how names that are the subject of the document and how variables / arguments should be typeset, but this doesn't mean the emphasis is suddenly disallowed.

SYNOPSIS is not as free flow, as it's meant as a summary.  We can be strict there.

Reply:
There are no issues with the script right now.  If someone feels a crying need to do ```B<thisI<really>word>``` and doc-nits complains, we can fix the script.

Optimize the common case: it's much more common to be an error.

Reply:
Actually, I would say that there *is* an issue with this script right now.  It had you remove an emphasis in `doc/man3/BIO_ADDR.pod`, which is *actually perfectly fine* in that description.  I'm not terribly attached to that emphasis, but it does serve as an example where this script *does too much*.

I cannot see myself approving this on that ground.

Reply:
I think the emphasis was (being polite) incorrect. Again, you think this does too much and I think it catches things that are more likely to be wrong.

Sorry you're not going to approve (after you've done all that review work), but that happens.  I think optimizing the common case -- the use is wrong -- is more important.  Shrug.

Someone else can take this PR as the basis for their own PR, of course.